addon-test-support/helpers.js

```diff
@@ -0,0 +1,7 @@
+import { find, scrollTo } from '@ember/test-helpers';
+
+export function scrollToPercentage(selector, percentage) {
+  const el = find(selector);
+  const y = ((el.scrollHeight - el.clientHeight) / 100) * percentage;
+  return scrollTo(el, 0, y);
```

Comment:
If there is no scrollbar, then there is no need to use `scrollTo`.
So, Is it okay to have `scrollTo` ?

I think we should check for scrollbar then use `scrollTo`.

Reply:
I don't think that's necessary?